### PR TITLE
add bertly to MEL

### PR DESCRIPTION
**Pull request by sohaibhasan**
* This PR adds bertly link clicks to the member event log so we can start tracking them in MAM counts and action count breakdowns

Ticket: https://www.pivotaltracker.com/story/show/158126302

### CODE REVIEW

File: Infrastructure/Scripts/derived_tables/MEL_v3.sql [hunk 1]
```diff
@@ -148,6 +148,16 @@ CREATE MATERIALIZED VIEW public.member_event_log AS
             pe.event_name IN ('share action completed', 'facebook share posted')
         AND pe.northstar_id IS NOT NULL
         AND pe.northstar_id <> ''
+	UNION ALL 
+    		SELECT -- SMS LINK CLICKS FROM BERTLY
+    			b.northstar_id AS northstar_id,
+    			b.click_time AS "timstamp",
```
Comment (jamjensen) [inline Infrastructure/Scripts/derived_tables/MEL_v3.sql]: missing the "e" in "timestamp"

Comment (sohaibhasan) [reply Infrastructure/Scripts/derived_tables/MEL_v3.sql]: Yikes! Good catch

**Comment by sohaibhasan** [comment]
@jamjensen Updated to PR to fix the spelling error

**Comment by jamjensen** [comment]
Not obviously super pressing -- but spacing is an additional tab to the right more than the rest of the UNIONs. 

We've also included a DISTINCT on all northstar_id. When reflecting on that I'm not sure if it's actually adding any value to the query, but I think we included it at the time because we saw duplicate records.

**Comment by sohaibhasan** [comment]
Think the tabbing is a dbeaver issue. I see it in sublime but not in dbeaver. Fixed regardless.
Couldn't hurt to have the distinct in there so why not. Update incoming

**Pull request merged**